common/reference/baseAddToGlobal.js

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
That was my first attempt, it was failing. But I just realized why, this code was being applied to the last statement in `base.js`
```javascript
BenchmarkSuite.prototype.RunStep = function(runner) {
  ...
}
```
which called  `function(runner) {...}` with `(functiont() { g = ...})` as parameter for `runner`. That is why the new version will have stray `;` at the beginning (assuming `base.js` is not modified):
```javascript
;(function(g) {
  g.performance = performance;
  g.Benchmark = Benchmark;
  g.BenchmarkSuite = BenchmarkSuite;
})((typeof global === "object" && global && global["Object"] === Object) ? global : this);
```
I will add a comment in this file as whell.